src/components/Work/Tabs/Cite.js

```diff
@@ -4,36 +4,39 @@ import MetadataDisplay from "../MetadataDisplay";
 
 const styles = {
   monoSpace: {
-    fontFamily: "monospace"
+    fontFamily: "monospace",
   },
   tabContent: {
-    padding: "0 1rem"
-  }
+    padding: "0 1rem",
+  },
 };
 
-const TabsCite = props => {
+const TabsCite = ({ item }) => {
   const {
     admin_set: { title: [admin_set] = "" } = "", // division,
-    collection = null,
-    date: [date] = "",
+    collection,
+    createDate: [date] = "",
+    descriptiveMetadata,
     id = "",
-    identifier = null,
-    license = null,
     nulUseStatement = "",
-    title: title = ""
-  } = props.item;
+  } = item;
+
+  const {
+    identifier = "",
+    license = null,
+    title: title = "",
+  } = descriptiveMetadata;
 
   const nul = "Northwestern University Libraries";
   const item_link = `${window.location.origin}/items/${id}`;
```

Comment:
Mind if we start eliminating the "snake case" titles of variables where you see them?